documents-storage-jcr/src/test/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorageTest.java

```diff
@@ -190,6 +190,17 @@ public void shareDocument() throws Exception {
     //assert that the linkNode set edit permission
     verify(linkNode).setPermissions(argThat((Map<String, String[]> map) -> map.containsKey("username") && Arrays.equals(map.get("username"),new String[]{"edit"})));
 
+    when(rootNode.hasNode("Shared")).thenReturn(true);
+    when(rootNode.getNode("shared")).thenReturn(sharedNode);
+    when(sharedNode.hasNode(currentNode.getName())).thenReturn(true);
+    when(sharedNode.getNode(currentNode.getName())).thenReturn(linkNode);
+    when(linkNode.getACL()).thenReturn(acl);
+
+    jcrDocumentFileStorage.shareDocument("1", 1L);
+    // Assert that the shared document event was not broadcast
```

Comment:
I don't get it, the broadcast is done 2 times, but the comment says that it should not have been fired.

Reply:
The broadcast is performed twice in the previous test case. PR updated !